templates/visuals/next/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "allowJs": false,
+        "emitDecoratorMetadata": true,
+        "experimentalDecorators": true,
+        "target": "ES6",
+        "sourceMap": true,
+        "outDir": "./.tmp/build/",
+        "sourceRoot": "../src/",
```

Comment:
This is no longer required since related issue has been fixed in PBIVIZ 1.13.0 by Excel team.